jetty-core/jetty-http2/jetty-http2-hpack/src/main/java/org/eclipse/jetty/http2/hpack/HpackException.java

```diff
@@ -29,9 +34,31 @@ public abstract class HpackException extends Exception
      */
     public static class StreamException extends HpackException
     {
-        public StreamException(String messageFormat, Object... args)
+        private final boolean request;
+        private final boolean response;
+
+        public StreamException(boolean request, boolean response, String messageFormat, Object... args)
```

Comment:
Why are these exceptions taking format + args instead of a pre-formatted string?